impact.tex

```diff
@@ -57,7 +54,24 @@ \subsection{Pathway toward impact}
 \end{draft}
 
 
-\TOWRITE{}{Start with brief narrative summary, hitting 'Scale and Significance' hard. We have close to nothing on Significance, yet. Scale could be clearer: All researchers!}
+Almost all researchers perform some kind of computation in the course of their work,
+from large-scale simulations to production of simple charts and figures.
+Researchers today are faced with a great variety of useful
+and increasingly open tools to perform their research tasks.
+However, this variety presents its own challenge.
+Because we are building and extending \textbf{generic tools for computational reproducibility}.
+We are not solving the \emph{whole reproducibility problem},
+but we \emph{are} solving a big piece of the problem (\textbf{reproducible computational environments}) in a way that can be reused,
+and mixed and matched with any number of solutions to other parts of the reproducibility problem,
+such as workflow execution.
+This reuse means other projects can focus on their part of the problem,
+without reimplementing a solution of software environments,
+improving the impact and efficiency of those projects, in turn.
```

Comment:
Shoudl we mention "interoperability" here. I think this is what you are referring to here.

Reply:
Yes, I struggled to come up with a good way to express this. This and the previous sentence are trying to hit two related points. I want to communicate that we are solving a universal piece of the reproducibility process in a way that can be re-used by others who solve a _differerent_ piece of the puzzle. I want to signal two benefits:

1. broad applicability of our work (lots of people can benefit, beyond all examples in our proposal; big impact for us), and
2. if we do a good job, others can focus on their own problem, without having to solve the problem we are solving again, which makes them more efficient, more effective, etc.


Want to have a go at communicating that better?

Reply:
I am not sure this is exactely what you want to say and it is a bit "heavy":

By focusing on (\textbf{reproducible computational environments}), and offering a generic and interoperable solution, we \emph{are} solving a big piece of the reproducibility problem and act as a bridge between any solutions to other parts of the reproducibility problem, for instance those classicaly addressed by workflow management systems. "Do one thing and do it well" is the key to success and will accelerate progress by allowing others to concentrate on other challenges of the reproducibility problem.